gbfs-validator/gbfs.js

```diff
@@ -303,38 +303,46 @@ class GBFS {
 
   getFile(type, required) {
     if (this.autoDiscovery) {
-      const urls = Object.entries(this.autoDiscovery.data).map(key => {
-        return Object.assign(
-          { lang: key[0] },
-          this.autoDiscovery.data[key[0]].feeds.find(f => f.name === type)
-        )
-      })
+      let urls
+
+      let version = this.options.version || this.autoDiscovery.version
+
+      if (version === '3.0-RC') {
+        urls = this.autoDiscovery.data.feeds?.filter(f => f.name === type) || []
```

Comment:
suggestion: If this is the new pattern for extracting the URLs from version 3+. I suggest having a more generic conditional to avoid refactoring when new versions are available. Maybe something like: 
```
if (  ( Number.parseInt( (version || "0" ) .split(".")[0])) >= 3)  ( 
```

Reply:
With the merge of the "3.0" schemas (not RC), a change was indeed necessary. 
I've updated the code to:
```js
// 3.0-RC , 3.0 and upcoming minor versions
if (/^3\.\d/.test(version)) {
```